wallet/src/de/schildbach/wallet/ui/dashpay/CreateIdentityService.kt

```diff
@@ -520,6 +521,19 @@ class CreateIdentityService : LifecycleService() {
 
         finishRegistration(blockchainIdentity, encryptionKey)
 
+        invite?.apply {
+            val results = platformRepo.getUser(user)
```

Comment:
In order to call this method the `platformRepo` needs to be initialized. That is why I changed the `platformRepo.init()` to block the execution.